### PR TITLE
Add trigger and rule to OpenWhisk Client

### PR DESCRIPTION
**Pull request by YujiAzama**
Added Trigger and Rule implementation to OpenWhisk Client for Schedules API.

### CODE REVIEW

File: meteoroid/core/tests/libs/clients/test_open_whisk_client.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
+import unittest
+from unittest.mock import Mock
+from unittest.mock import patch
+
+from ....libs.clients.open_whisk_client import OpenWhiskClient
+from ....libs.clients.open_whisk_client import OpenWhiskClientException
+
+
+class MockResponse(object):
```
Comment (hide-hayashi) [inline meteoroid/core/tests/libs/clients/test_open_whisk_client.py]: the class doesn't use?

Comment (YujiAzama) [reply meteoroid/core/tests/libs/clients/test_open_whisk_client.py]: Thank you for your comment.
MockResponse is not used. It has been deleted.

**Review by hide-hayashi — APPROVED** [review]
Thank you for fixed code. LGMT .

@ryo-ma  if you finish review ,  you can merge this code.

File: meteoroid/core/libs/clients/open_whisk_client.py [hunk 2]
```diff
@@ -139,3 +139,73 @@ def retrieve_activation_logs(self, activation_id, namespace):
...
+        return response
+
+    def create_rule(self, namespace, data):
+        self.headers['Content-Type'] = 'application/json'
+        rule_name = data.get('name')
+        response = requests.put(f'{self.endpoint}/api/v1/namespaces/{namespace}/rules/{rule_name}',
+                                 headers=self.headers,
+                                 data=json.dumps(data),
+                                 auth=(self.user, self.password),
+                                 verify=False)
```
Comment (ryo-ma) [inline meteoroid/core/libs/clients/open_whisk_client.py]: Continuation line over-indented for visual indent.
Please delete spaces.

I think others are good.

**Review by ryo-ma — COMMENTED** [review]
Please make sure a following review.

File: meteoroid/core/libs/clients/open_whisk_client.py (hunk 2 above)
Comment (YujiAzama) [reply meteoroid/core/libs/clients/open_whisk_client.py]: Well noticed! Thank you.

**Comment by YujiAzama** [comment]
I updated.

**Review by ryo-ma — APPROVED** [review]

**Pull request merged**